javaslang-benchmark/src/test/java/javaslang/collection/VectorBenchmark.java

```diff
@@ -578,6 +581,7 @@ public void slang_persistent(Blackhole bh) {
             for (int i = 1; !values.isEmpty(); i++) {
                 values = values.slice(1, values.size());
                 values = values.slice(0, values.size() - 1);
+                assert sizeOf(values.trie) < (branchingFactor() * sizeOf(copyRange(ELEMENTS, i, javaMutable.size() - i))); /* detect memory leak */
```

Comment:
added leak detection, i.e. if `slice` only keeps a few elements from a lot, its size should be small